nestkernel/model_manager.cpp

```diff
@@ -460,6 +460,14 @@ ModelManager::connector_requires_symmetric( synindex syn_id ) const
   return prototypes_[ 0 ][ syn_id ]->requires_symmetric();
 }
 
+bool
+ModelManager::connector_requires_clopath_archiving( synindex syn_id ) const
```

Comment:
here and in similar places add `const` to parameter, please check all other functions you added

Reply:
Done!